examples/MQ/histogramServer/FairMQExHistoDevice.cxx

```diff
@@ -55,7 +55,7 @@ bool FairMQExHistoDevice::ConditionalRun()
     auto message(NewMessage());
     RootSerializer().Serialize(*message, &fArrayHisto);
 
-    for (auto& channel : fChannels) {
+    for (auto& channel : GetChannels()) {
         Send(message, channel.first);
```

Comment:
Using the new API unconditionally means you require FairMQ v1.5.0. Is this intended? If yes, then we should express this in the `find_package` call.